### DISCUSSION TITLE
Github search/code api doesnot gives results if repo owner has digit in its name.

### DISCUSSION THREAD

**Discussion by Jay8181** [community/community#56404] [Apps, API and Webhooks]
### Select Topic Area

Bug

### Body

Github "search/code" api does not gives results if repo owner has digit in its name.

**Comment by Jay8181** [comment]
Repository Where the owner name does not contain any digits in it.(23 results are shown)
![image](https://github.com/community/community/assets/80192066/967e6e1a-da59-4258-a278-b50594d929a2)

Fork of the repository where the owner name consists of digits.(0 results shown)
![image (1)](https://github.com/community/community/assets/80192066/07b1213a-9d37-4352-85f6-879f8686bba7)

**Comment by 3V3RYONE** [comment]
I can confirm this issue.

**Reply by Jay8181** [reply]
Thanks @3V3RYONE .

**Reply by Jay8181** [reply]
Were you able to replicate the issue ???
